[2023-02-19 12:00:43,471] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: members_delta_hourly_extract.validate_task manual__2023-02-19T12:00:36.598018+00:00 [queued]>
[2023-02-19 12:00:43,483] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: members_delta_hourly_extract.validate_task manual__2023-02-19T12:00:36.598018+00:00 [queued]>
[2023-02-19 12:00:43,485] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-02-19 12:00:43,498] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2023-02-19 12:00:43,501] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-02-19 12:00:43,521] {taskinstance.py:1377} INFO - Executing <Task(MembersDeltaHourlyValidateOperator): validate_task> on 2023-02-19 12:00:36.598018+00:00
[2023-02-19 12:00:43,534] {standard_task_runner.py:52} INFO - Started process 17370 to run task
[2023-02-19 12:00:43,540] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'members_delta_hourly_extract', 'validate_task', 'manual__2023-02-19T12:00:36.598018+00:00', '--job-id', '58', '--raw', '--subdir', 'DAGS_FOLDER/members_delta_extract_hourly_dag/members_delta_hourly_extract.py', '--cfg-path', '/tmp/tmp4zz3ae5h', '--error-file', '/tmp/tmpl04y94uz']
[2023-02-19 12:00:43,543] {standard_task_runner.py:80} INFO - Job 58: Subtask validate_task
[2023-02-19 12:00:43,620] {task_command.py:369} INFO - Running <TaskInstance: members_delta_hourly_extract.validate_task manual__2023-02-19T12:00:36.598018+00:00 [running]> on host 2eb478c84c88
[2023-02-19 12:00:43,708] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=members_delta_hourly_extract
AIRFLOW_CTX_TASK_ID=validate_task
AIRFLOW_CTX_EXECUTION_DATE=2023-02-19T12:00:36.598018+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-02-19T12:00:36.598018+00:00
[2023-02-19 12:00:43,710] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-02-19 12:00:43,752] {members_delta_hourly_validate.py:30} INFO - There are files to validate
[2023-02-19 12:00:43,755] {members_delta_hourly_validate.py:65} INFO - Start Validating Data
[2023-02-19 12:00:43,777] {members_delta_hourly_validate.py:91} INFO - All validation checks have passed
[2023-02-19 12:00:43,797] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=members_delta_hourly_extract, task_id=validate_task, execution_date=20230219T120036, start_date=20230219T120043, end_date=20230219T120043
[2023-02-19 12:00:43,880] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-19 12:00:43,922] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
